### ISSUE TITLE
Integrate with website

### ISSUE THREAD

**Issue by locks** [ember-learn/builds#5]
Figure out how to best integrate it with the website repo for publishing.

**Comment by acorncom** [comment]
The Ember website is published by this script: https://github.com/emberjs/website/blob/master/.travis/deploy.sh

As a result, we're likely going to need to do a push of the built files from this repo to the appropriate spot on the Ember website repo and then do a commit there.  Could get a bit tricky ...  These ideas are open for discussion / suggestion 😄

**Comment by sivakumar-kailasam** [comment]
@acorncom @locks I got website working when I did these,
1. Minor modifications to ember-cli-builds.js to remove config meta tag lookup, appending hash to minified files(production builds).
2. Replace contents of https://github.com/emberjs/website/tree/master/source/javascripts/app/builds/ with the vendor.js and build.js files created by this app.
3. modify https://github.com/emberjs/website/blob/master/source/javascripts/builds.js to
```
//= require ./app/builds/vendor
//= require ./app/builds/builds
```

Until website and this app become ember engines, we can keep publishing these artifacts to either github releases/s3 and include them in the website builds. Downside being that all styles & static assets need to get added to both repos until the engines come along. Let me know your thoughts on the same, based on that I can raise a PR to both repos.
